### PR TITLE
Fire - Fix animals screaming like humans

### PR DESCRIPTION
**Pull request by Whigital**
**When merged this pull request will:**
- Only play screams if unit on fire is a human

Though hilarious, not too immersive and in line the realism ACE strives for ....

### CODE REVIEW

**Review by PabstMirror — APPROVED** [review]

**Review by jonpas — APPROVED** [review]
@veteran29 Can you make a new issue for that? I'd like to see that too, but it doesn't have to block this PR.

**Comment by veteran29** [comment]
> @veteran29 Can you make a new issue for that? I'd like to see that too, but it doesn't have to block this PR.

I can fix it in this PR if the author (@Whigital is fine with this).

**Comment by Whigital** [comment]
@veteran29 No worries, go ahead 👍

**Review by PabstMirror — APPROVED** [review]

File: addons/fire/functions/fnc_burnReaction.sqf [hunk 1]
```diff
@@ -5,23 +5,27 @@
...
  */
 
-params ["_unit"];
+params ["_unit", ["_throwWeapon", true]];
 
 if (
-    GVAR(dropWeapon) > 0
+    _throwWeapon
+    && {GVAR(dropWeapon) > 0}
     && {_unit in _unit && { !(currentWeapon _unit isEqualTo "") }}
```
Comment (PabstMirror) [inline addons/fire/functions/fnc_burnReaction.sqf:21]:
```suggestion
    && {_unit in _unit && {(currentWeapon _unit) != ""}}
```

Comment (PabstMirror) [reply addons/fire/functions/fnc_burnReaction.sqf:21]: this is unrelated to this PR, so lets just do it later

Comment (PabstMirror) [reply addons/fire/functions/fnc_burnReaction.sqf:21]: also maybe check hitreactions exists

**Pull request merged**